packages/fgtclb/academic-jobs/Configuration/Sets/AcademicJobs/settings.definitions.yaml

```diff
@@ -2,42 +2,67 @@ categories:
   academic:
     label: 'Academic'
   academic.jobs:
-    label: 'Persons'
+    label: 'Jobs'
```

Comment:
Is it not possible to use translation labels here (and the whole file) ? (`LLL:...`), I would say that would be a major oversight in TYPO3 if that would not work.

We should abstain from using hardcoded valued for things like labels if translation is possible !!!!

Reply:
Talkend direct, we take this now like done and use translation labels in all extensions and files as dedicated change(s) afterwards.